client/components/toggle/style.scss

```diff
@@ -0,0 +1,9 @@
+.toggle__text {
+	color: $gray;
+	font-size: 11px;
+	line-height: 16px;
+	margin-left: 8px;
+	vertical-align: middle;
+	text-transform: uppercase;
+	cursor: pointer;
```

Comment:
Removed pointer in https://github.com/Automattic/woocommerce-connect-client/pull/376/commits/77e6c33e877d6931f17e4958e855171bc1cf814b